index.rst

```diff
@@ -44,18 +44,589 @@
+
+- The origin of M1M3 CS overlaps with OCS origin, i.e., at the theoretical vertex of M1.
+- +x points toward actuator 106.
+- +y points toward actuator 441, which is close to the M1M3 mirror cell door.
+- +z points toward the sky.
+- **When mounted on the TMA, M1M3 CS is the same as OCS.**
+
+
+.. figure:: /_static/m1m3.png
+   :name: fig-m1m3
+   :target: ../_images/m1m3.png
+   :alt: The M1M3 CS
+
+The M1M3 CS.
+
+Our goal here is not to change all the engineering drawings to be in this CS. Instead, the goal is to make sure that for anything that is being used by the AOS, we can put them into M1M3 CS or OCS correctly.
+
+Note that M3 vertex is at (0, 0, -233.8)mm in the OCS.
+
+The Rubin Obs. official M1M3 Finite Element Model (FEM), as provided by Doug Neil and Ed Hileman, uses the M1M3 CS.
+`The bending mode shapes and forces derived using this FEM
+<https://github.com/lsst-sitcom/M1M3_ML/blob/master/data/M1M3_1um_156_README.txt>`__
+use the M1M3 CS as well.
+
+- When the force on an single-axis actuator or the primary cylinder of a lateral or bilateral actuator is positive, it pushes M1M3 toward the sky, along +z axis. The bending mode forces are given `here <https://github.com/lsst-sitcom/M1M3_ML/blob/master/data/M1M3_1um_156_force.txt>`__.
+- For bending modes, there are two variaties. The `surface normal bending modes <https://github.com/lsst-sitcom/M1M3_ML/blob/master/data/M1M3_1um_156_grid.txt>`__ are those that were directly measured in the RFCML using the interferometers. Here the displacement vectors of the Finite Element nodes point toward the center of curvature, and are normal to the M1M3 surface. For use in an optical raytrace program like Zemax or PhoSim, and for deriving the senM, we need the `surface sag bending modes <https://github.com/lsst-sitcom/M1M3_ML/blob/master/data/M1M3_1um_156_sag.txt>`__. These displacement vectors point along +z axis of the OCS or M1M3 CS.
+
+Like other components of the AOS, M1M3 operates mostely off its Look-Up Table (LUT), which contains our best knowledge of the forces as functions as gravity (or zenith angle) and temperature profiles on and around the mirror surfaces. The current M1M3 LUT can be found `here <https://github.com/lsst-sitcom/M1M3_ML/blob/master/data/FLUT.yaml>`__.
+
+- The zenith angle, as the primary input to the M1M3 LUT, is defined the same way as the OCS zenith angle as defined in Sec. :ref:`section-ocs`.
+- Unrelated to the bending modes, but relevant to the LUT, are the forces on the secondary cylinders of the lateral and bilateral actuators. When the force on the secondary cylinder of an lateral actuator is positive, it pushes M1M3 in the y-z plane, along 45 degrees between +y and +z axes. When the force on the secondar cylinder of a bilateral actuator is positive, it pushes M1M3 in the x-z plane, along the 45 degree line between either the +z and +x or the +z and -x directions, depending on the location of the bilateral actuator.
```

Comment:
Can you specify which bilateral actuators do "+z and +x" or "+z and -x" in M1M3?